source/slang/slang-ir-insts.h

```diff
@@ -2842,6 +2842,14 @@ struct IRGlobalVar : IRGlobalValueWithCode
 struct IRGlobalParam : IRInst
 {
     IR_LEAF_ISA(GlobalParam)
+
+    /// Get the entry point if this global param originates from an entry point parameter,
```

Comment:
Instead of extending global param with a new operand, I think it is better to add a new decoration on the global param, so that global params that is not coming from EntryPoint does not need to carry a null operand.